R/caretStack.R

```diff
@@ -43,6 +49,8 @@ caretStack <- function(
     metric = NULL,
     trControl = NULL,
     excluded_class_id = 1L,
+    use_original_features = FALSE,
+    which.var = NULL,
```

Comment:
Can you combine these 2 parameters into one? So instead of `use_original_features` and `which.var`

You have `original_features` with a default `original_features=NULL` or `original_features=c()` which means use none of them by defauly

Reply:
This would simplify the function. I am going to apply this change. Thanks